server/api.go

```diff
@@ -29,6 +28,9 @@ const (
 	upBoundCost = 16
 )
 
+// version is the version of the Dex server. It is injected during build stage.
+var version = "was not built properly"
```

Comment:
I'm wondering if injecting the version instead would make more sense.

Reply:
We can inject the version variable at build-time only to the main package and then do something like

cmd
```go
import "github.com/dexidp/dex/server"

var version = "DEV"

func init() {
  server.Version = version
}
```

server
```go
var Version = "DEV"
```
Is it ok? It looks like it will work as we want.

Reply:
I rather meant injecting into the server.

Something like this:

```go
NewAPI(storage, version, logger)
```

Reply:
Not sure if that's a good idea, but global stuff always makes me icky.